Chapter1/Array_Functions.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+namespace BootCamp.Chapter1
+{
+    public static class Array_Functions
+    {
+        public static int[] Sort_Array(int[] array)
+        {
+            int temp;
+            for (int j = 0; j <= array.Length - 2; j++)
```

Comment:
Why do both I and j have the same end? It means that you compare the same number twice.

Reply:
Lets say we have the array [ 4 2 3 5].

I goes then from 4 till the 5 
J goes then from 2 tilll the 5 
and then from 3 5 
and then only the 5 

So all numbers after a certain number are compared to that number.